HALO/wales/wales_wv.yaml

```diff
@@ -0,0 +1,95 @@
+plugins:
+  source:
+    - module: intake_xarray
+sources:
+  HALO-0122:
+    description: WALES-Lidar Water Vapour Mixing Ratio in g/kg.
```

Comment:
Thanks for updating the description in the main catalog. However, the description in this file now doesn't match anymore with the main file. Will you update it here as well?

Reply:
🙈